.github/CODEOWNERS

```
@@ -1 +1,2 @@
 *             @gravitee-io/techops @jgiovaresco
+*             @gravitee-io/devops
```

Maybe we can only keep one line:

```suggestion
*             @gravitee-io/devops
```

---

done